### DISCUSSION TITLE
Tortuosity smaller than 1?

### DISCUSSION THREAD

**Discussion by dion-w** [pybamm-team/PyBaMM#1712] [Q&A] [answered]
Hello everyone,

i got confused when i recently looked into the code and found that your calculated tortuosities are below 1. As i understood  the concept of tortuosity (or more precisely "tortuosity factor" coming from [this paper](https://doi.org/10.1016/j.coche.2016.02.006)) this property represents the effective transport length in the conductive phase of a medium with obstacles in it. A Tortuosity factor of 1 represents the shortest possible connection between two points, whereas a tortuosity bigger than 1 is a pathway with obstacles in it. 

I think your equations are still correct (i don't know for sure), but it's very confusing to see the property you call "tortuosity" being smaller than 1, as this would mean the transport pathway in your electrode is even shorter than a direct connection. 

My suggestion would be to change the name of the variable to something like "effective transport coefficient" to not confuse it with the concept of tortuosity as a lot of people in the battery community work with tortuosities bigger than 1.

So what do you think about it? 

I'm coming from the battery community and my knowledge is mostly based on my research at [my institute](https://www.iam.kit.edu/et/english) as well as [this paper](https://iopscience.iop.org/article/10.1149/2.0731814jes) and [this publication](https://doi.org/10.1016/j.coche.2016.02.006). Contact details about me can be found [here](https://www.iam.kit.edu/et/english/mitarbeiter_5143.php). 

Best regards,
Dion Wilde

**Comment by valentinsulzer** [comment, marked as answer]
Happy to have this change, what do you think @brosaplanella ?

**Reply by brosaplanella** [reply]
Yup, I agree with Dion. I think "transport efficiency" would be a better name: it is a number between 0 and 1 which is the ratio between the actual value and the ideal one (bulk material).

**Reply by valentinsulzer** [reply]
Cool, @dion-w can you open a PR to do this? Should basically be a repo-wide find-and-replace from "tortuosity" to "transport efficiency", including some file names

**Reply by dion-w** [reply]
@tinosulzer  i'll see if i can do the PR this week. There's still some other stuff to do but should be fine.
